interpreter/meta/travis/install-ocaml.sh

```diff
@@ -2,26 +2,45 @@
 
 set -e
 
+download_from_gh_archive() {
+  local project=$1;
+  local version=$2;
+  local sha=$3;
+
+  curl https://github.com/ocaml/${project}/archive/${version}.tar.gz -OL
+  CHECKSUM=$(shasum -a 256 ${version}.tar.gz | awk '{ print $1 }')
+  if [ ${CHECKSUM} != ${sha} ]; then
+    echo "Bad checksum ${project} download checksum!"
+    exit 1
+  fi
+  tar xfz ${version}.tar.gz
+}
+
 # Move to a location relative to the script so it runs
 # from anywhere. Go three levels down to get out of interpreter/
 # and into the top-level dir, since we'll run ocamlbuild
 # inside of interpreter/ and it goes pear-shaped if it
 # encounters ocaml's own build directory.
 cd $(dirname ${BASH_SOURCE[0]})/../../..
 
+PREFIX=$PWD/ocaml/install
+
 rm -rf ocaml
 mkdir ocaml
 cd ocaml
-curl https://wasm.storage.googleapis.com/ocaml-4.02.2.tar.gz -O
-CHECKSUM=$(shasum -a 256 ocaml-4.02.2.tar.gz | awk '{ print $1 }')
-if [ ${CHECKSUM} != \
-  9d50c91ba2d2040281c6e47254c0c2b74d91315dd85cc59b84c5138c3a7ba78c ]; then
-  echo "Bad checksum ocaml download checksum!"
-  exit 1
-fi
-tar xfz ocaml-4.02.2.tar.gz
-cd ocaml-4.02.2
-./configure -prefix $PWD/../install
+mkdir install
+
+download_from_gh_archive ocaml 4.05.0 e5d8a6f629020c580473d8afcfcb06c3966d01929f7b734f41dc0c737cd8ea3f
+cd ocaml-4.05.0
+./configure -prefix ${PREFIX}
 make world.opt
-mkdir ../install
+make install
+cd ..
+
+PATH=${PREFIX}/bin:${PATH}
+
+download_from_gh_archive ocamlbuild 0.11.0 1717edc841c9b98072e410f1b0bc8b84444b4b35ed3b4949ce2bec17c60103ee
+cd ocamlbuild-0.11.0
+make configure
+make
 make install
```

Comment:
Do you need `OCAMLBUILD_PREFIX=${PREFIX}` here or does it pick that up from ocaml itself?

Reply:
It pulls it from ocaml. In fact, I tried setting it manually but it didn't seem to work properly.

Reply:
Odd, that's what I did in https://github.com/WebAssembly/waterfall/pull/244

Reply:
Hm, I probably screwed something up. :-) Oh well.